imagersite/imager_profile/tests.py

```diff
@@ -60,3 +80,46 @@ def test_delete_profile_deletes_user(self):
         self.assertEqual(len(ImagerProfile.objects.all()), 1)
         user.profile.delete()
         self.assertEqual(len(User.objects.all()), 0)
+
+
+class ProfileViewTestCase(TestCase):
+    def setUp(self):
+        user = UserFactory(username='userbob', email='bob@example.com')
+        user.set_password('secret')
+        user.save()
+        user.profile.camera = 'Canon'
+        user.profile.address = '123 Bob Drive'
+        user.profile.website = 'www.bob.com'
+        user.profile.photography_type = 'portraits'
```

Comment:
might be nice to have a method on the profile class that takes these keyword arguments so you can call a single method with a dictionary of key/val pairs instead. It might even come in handy in editing the profile later on.